test/view_models/paye/PayeGovernmentSpendSpec.scala

```diff
@@ -57,12 +59,21 @@ class PayeGovernmentSpendSpec extends UnitSpec with MockitoSugar with JsonUtil w
         )
         val result = PayeGovernmentSpend(payeGovSpendingData)
 
-        result.orderedSpendRows.size shouldBe PayeGovernmentSpend.orderedSpendCategories.size
+        result.orderedSpendRows.map(_._2.percentage) shouldBe PayeAtsTestData.expectedPercentageOrder
         result shouldBe PayeAtsTestData.payeGovernmentSpendViewModel.copy(
           isScottish = false
         )
       }
 
     }
+
+    "sort transport above Public Order" in {
+
```

Comment:
Is this tested in the backend? Not sure if you require a test here as it's basically whatever backend gives you?

Reply:
Backend is sorting the data based on percentage. But if percentage of transport and Public Order is same then we have to show transport above the public order so this reordering is done on frontend hence this test case.